cmd/bbolt/main.go

```diff
@@ -2024,5 +2032,13 @@ Additional options include:
 	-tx-max-size NUM
 		Specifies the maximum size of individual transactions.
 		Defaults to 64KB.
+
+	-no-sync BOOL
+		Skip fsync() calls after each commit (fast but unsafe)
+		Defaults to false
+
+	-no-freelist-sync BOOL
+		Skip syncing freelists to disk (fast but unsafe)
```

Comment:
This is safe. The downside is to scan the entire database to re-compute the free list at the startup time.

Reply:
(If you use this together with NoSync, it is unsafe. But the root cause is the NoSync flag, not this one)

Reply:
Should I remove this option?

Reply:
What does 'safe' mean in this context? If you mean safe from corruption due to kernel I/O errors, I'm so super worried in this context, that's unlikely right, like a power failure or someone unplugs the disk?

The compact command can either succeed or fail, if the target db is corrupt for any reason we can just fail loudly with an error.